lib/forms_tests.js

```diff
@@ -0,0 +1,278 @@
+        expect(err).to.not.exist;
+        expect(data).to.be.an('object');
+        expect(data.forms).to.be.an('Array');
+        expect(data.forms.length).to.equal(1);
+        expect(data.forms[0]).to.be.an('object');
+        expect(data.forms[0].name).to.equal(definitions.form.name);
+        expect(data.forms[0].createdBy).to.equal(definitions.form.createdBy);
+        expect(data.forms[0]._id).to.equal(definitions.form._id);
+        formId = data.forms[0]._id;
+        done();
+      } catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('should get form', function(done) {
+    fh.forms.getForm({ _id: formId }, function(err, data) {
+      try {
+        expect(err).to.not.exist;
+        expect(data.name).to.equal(definitions.form.name);
+        expect(data.createdBy).to.equal(definitions.form.createdBy);
+        expect(data._id).to.equal(definitions.form._id);
+        expect(data.pages.length).to.equal(definitions.form.pages.length);
+        expect(data.pages[0].fields.length).to.equal(definitions.form.pages[0].fields.length);
+        formDefinition = data;
+        done();
+      } catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  // https://issues.jboss.org/browse/RHMAP-16402
+  // it('should get populated form list', function(done) {
+  //   fh.forms.getPopulatedFormList({ formids: [formId] }, function(err, data) {
+  //     try {
+  //       expect(data).to.be.an('Array');
+  //       expect(data.length).to.equal(1);
+  //       expect(data[0]).to.eql(formDefinition);
+  //       done();
+  //     } catch (e) {
+  //       done(e);
+  //     }
+  //   });
+  // });
+
+  it('should get theme', function(done) {
+    fh.forms.getTheme({}, function(err, data) {
+      try {
+        expect(err).to.not.exist;
+        expect(data).to.be.an('object');
+        expect(data.name).to.equal(definitions.theme.name);
+        expect(data.createdBy).to.equal(definitions.theme.createdBy);
+        expect(data._id).to.equal(definitions.theme._id);
+        done();
+      } catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('should get app client config', function(done) {
+    fh.forms.getAppClientConfig({}, function(err, data) {
+      try {
+        expect(err).to.not.exist;
+        expect(data).to.be.an('object');
+        done();
+      } catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('should get no submissions', getSubmissions);
+
+  it('should pass submission test', function(done) {
+    async.series([
+      registerEventListener,
+      submitFormData,
+      completeSubmission,
+      getSubmissions,
+      deregisterEventListener,
+      submitFormData
+    ], done);
+  });
+
+  it('should export csv', function(done) {
+    var queryParams = {
+      projectId: definitions.projectId,
+      submissionId: submissionData.submissionId,
+      formId: [definitions.form._id],
+      fieldHeader: definitions.form.pages[0].fields[0].name
+    };
+
+    fh.forms.exportCSV(queryParams, done);
```

Comment:
It would be good to verify results of this call. In case of PDF exports its complicated, but CSV is easier. Not necessarily in this PR, but maybe create follow-up JIRA, @jhellar wdyt ?